wireless_modules/middle_module/main.py

```diff
@@ -16,27 +19,44 @@
 # Define Voltage divider factor for this module or leave as None to use default voltage factor
 VOLTAGE_FACTOR = None
 
-# Define all the Pin objects for each sensor
-scl_pin = machine.Pin(22)
-sda_pin = machine.Pin(21)
-dht_pin = machine.Pin(4)
-mq135_pin = machine.Pin(34)
-
-# Instantiate sensor objects
-my_mpu = Mpu(scl_pin, sda_pin, 20)
-my_mpu.calibrate()
-my_dht = DhtSensor(dht_pin)
-my_mq135 = CO2(mq135_pin)
-my_mq135.set_rzero(RZERO)
-
-battery_pin = 33
-battery_reader = BatteryReader(battery_pin, scale=1, voltage_factor=VOLTAGE_FACTOR)
-
-# Set up the wireless module
-middle_module = WirelessModule(MODULE_NUM, battery_reader)
-sensors = [my_mpu, my_dht, my_mq135]
-middle_module.add_sensors(sensors)
-
-# Enters an infinite loop
-print("starting loop")
-middle_module.run()
+
+async def main():
+    # Define all the Pin objects for each sensor
+    scl_pin = machine.Pin(22)
+    sda_pin = machine.Pin(21)
+    dht_pin = machine.Pin(4)
+    mq135_pin = machine.Pin(34)
+
+    # Instantiate sensor objects
+    # my_mpu = Mpu(scl_pin, sda_pin, 20)
+    # my_mpu.calibrate()
+    # my_dht = DhtSensor(dht_pin)
+    # my_mq135 = CO2(mq135_pin)
+    # my_mq135.set_rzero(RZERO)
```

Comment:
Just one small thing - I believe you forgot to uncomment these after testing 😂. Same goes for `line 42` below.

Reply:
Wow, thank you for picking this up haha